### PR TITLE
fixing edit URL and adding links for primers

### PR DESCRIPTION
**Pull request by psmulovics**

### CODE REVIEW

File: primers/artificial-intelligence/README.md [hunk 1]
```diff
@@ -1,91 +1,44 @@
...
-- [Amazon Web Services (AWS)](#_toc140607974)
-- [Google Cloud Platform (GCP)](#_toc140607975)
-- [Microsoft Azure](#_toc140607976)
-- [IBM Watson](#_toc140607977)
-- [Oracle Cloud Infrastructure (OCI)](#_toc140607978)
-
-## <a name="_toc140607934"></a>Introduction to Artificial Intelligence
+﻿---
+id: artificial-intelligence
+title: "Artificial Intelligence Primer"
```
Comment (polinalevyant) [inline primers/artificial-intelligence/README.md:3]: do we need "Primer" in the name? other two primers don't have it

Comment (polinalevyant) [reply primers/artificial-intelligence/README.md:3]: *Spatial computing doesn't have it, DLT does, so need to decide whether to add it for concistency

File: primers/blockchain-and-dlt/README.md [hunk 2]
```diff
@@ -1,56 +1,14 @@
...
-- [Algorand Blockchain](#_toc140607978)
-
-***
-
-## [Introduction to DLT](#_toc140607934)
+---
+title: "Decentralised Ledger Technology Primer (DLT)"
+---
+
+## Introduction to the Subject
```
Comment (polinalevyant) [inline primers/blockchain-and-dlt/README.md:5]: not in other primers, perhaps unnecessary

File: primers/spatial-computing/README.md [hunk 3]
```diff
@@ -1,80 +1,36 @@
...
-# [Introduction to Spatial Computing](#_Toc149764170)
-
-## [Definition & Overview](#_Toc149764171)
+---
+title: "Spatial Computing"
+---
+
+## Introduction to Spatial Computing
+
+### Definition & Overview
```
Comment (polinalevyant) [inline primers/spatial-computing/README.md:7]: ### for Introduction  and #### for Definition & Overview in other two primers

**Review by polinalevyant — APPROVED** [review]
some small formatting issues, but otherwise looks great!

File: primers/artificial-intelligence/README.md (hunk 1 above)
Comment (psmulovics) [reply primers/artificial-intelligence/README.md:3]: I'd say to drop it

File: primers/spatial-computing/README.md (hunk 3 above)
Comment (psmulovics) [reply primers/spatial-computing/README.md:7]: Good spot!

**Pull request merged**